src/ECharts/Color.purs

```diff
@@ -0,0 +1,31 @@
+module ECharts.Color where
+
+import Data.Function
+import Data.Argonaut.Core
+import Data.Argonaut.Encode
+import Data.Argonaut.Extension.Func
+
+import ECharts.Item.Value
+
+type Color = String
+
+
+
+
+newtype ColorFuncParam =
+  ColorFuncParam {
```

Comment:
This looks good, just make sure all these are always guaranteed to be non-null, or else they should be modeled with Maybe.